test/e2e/config/e2e_conf.yaml

```diff
@@ -167,7 +167,7 @@ intervals:
   default/wait-vm-state: ["20m", "100ms"]
   default/monitor-vm-state: ["1m", "500ms"]
   default/monitor-provisioning: ["5m", "500ms"]
-  default/wait-deployment: ["10m", "10s"]
+  default/wait-deployment: ["20m", "10s"]
```

Comment:
10 minutes is really not enough? :flushed:

Reply:
fixed